benchmarks/distributed/test_distributed.py

```diff
@@ -0,0 +1,20 @@
+import ray
+import ray.autoscaler.sdk
+
+from time import sleep
+
+MAX_NUM_NODES = 1000
+
+def scale_to(target):
+    ray.autoscaler.sdk.request_resources(bundles=[{"node": 1}] * target)
+    while len(ray.nodes()) != target:
+        sleep(5)
+
+def test_nodes():
+    scale_to(MAX_NUM_NODES)
+    assert len(ray.nodes()) == MAX_NUM_NODES
```

Comment:
We should also schedule a trivial task workload (bunch of sleeps), and check the cluster is fully utilized.

Reply:
i just added all the distributed tests